README.md

```diff
@@ -8,6 +8,8 @@
 [![Website](https://img.shields.io/website?url=https%3A%2F%2Foqt.ohsome.org)](https://oqt.ohsome.org)
 [![status: active](https://github.com/GIScience/badges/raw/master/status/active.svg)](https://github.com/GIScience/badges#active)
 
+Thanks to the [ohsome dashboard](https://ohsome.org/apps/dashboard/) generating quality estimations of OpenStreetMap data for an arbitrary region is now as easy as pie.
```

Comment:
```suggestion
Thanks to the [ohsome dashboard](https://dashboard.ohsome.org) generating quality estimations of OpenStreetMap data for an arbitrary region is now as easy as pie.
```

Reply:
Maybe move this to a `## Dashboard` section above `## Blog` and add a screenshot?

Reply:
And maybe add section about the new website, using the old link `https://oqt.ohsome.org`.